modules/localgov_campaigns_paragraphs/config/install/field.field.paragraph.localgov_block_view.localgov_embed_block_view.yml

```diff
@@ -0,0 +1,49 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.localgov_embed_block_view
+    - paragraphs.paragraphs_type.localgov_block_view
+  module:
+    - viewsreference
+id: paragraph.localgov_block_view.localgov_embed_block_view
+field_name: localgov_embed_block_view
+entity_type: paragraph
+bundle: localgov_block_view
+label: 'Block view'
+description: ''
+required: true
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:view'
+  handler_settings:
+    target_bundles: null
+    auto_create: 0
```

Comment:
Hi Gosia,
Could you please re-export this file after applying the patch.  Looks like the patched schema file is expecting *false* here instead of *0*.